android/build-scripts/publish-module.gradle

```diff
@@ -0,0 +1,65 @@
+apply plugin: 'maven-publish'
+apply plugin: 'signing'
+
+task androidSourcesJar(type: Jar) {
+    archiveClassifier.set('sources')
+    if (project.plugins.findPlugin("com.android.library")) {
+        from android.sourceSets.main.java.srcDirs
+    }
+}
+
+artifacts {
+    archives androidSourcesJar
+}
+
+group = PUBLISH_GROUP_ID
+version = PUBLISH_VERSION
+
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+                // The coordinates of the library, being set from variables that
+                // we'll set up later
+                groupId PUBLISH_GROUP_ID
+                artifactId PUBLISH_ARTIFACT_ID
+                version PUBLISH_VERSION
+
+                // Two artifacts, the `aar` (or `jar`) and the sources
+                artifact androidSourcesJar
+                artifact("$buildDir/outputs/aar/ouinet-release.aar")
```

Comment:
Hi @ivilata, I just realized that I'd probably need to change this line to support publishing artifacts from different architectures in the same package. I'll check it tomorrow and will push the changes to the same PR.

Reply:
Fixed in c316711. Now the artifacts will be published in Maven Central with the following structure: `ouinet-arm64-v8a:0.19.0`